packages/fxa-settings/src/models/Account.ts

```diff
@@ -776,7 +804,7 @@ export class Account implements AccountData {
   }
 
   async sendVerificationCode() {
-    await this.withLoadingStatus(
+    return this.withLoadingStatus(
```

Comment:
Q: What's this for? I found this reference, but it doesn't seem to be using the return value, so wasn't clear on the difference between `await` and `return` diff.

```js
// packages/fxa-settings/src/pages/Signup/Confirm/index.tsx#L152
await account.sendVerificationCode();
```

Reply:
Thanks for the catch, this change was from testing the return value. No longer needed.